tests/kafkatest/services/kafka/kafka.py

```diff
@@ -782,7 +782,8 @@ def prop_file(self, node):
 
         if self.use_new_coordinator:
             override_configs[config_property.NEW_GROUP_COORDINATOR_ENABLE] = 'true'
-    
+            override_configs[config_property.GROUP_COORDINATOR_REBALANCE_PROTOCOLS_CONFIG] = 'classic,consumer'
```

Comment:
just to double check, here we're adding both as a default, but still respecting any specific value that a test may have right? (I guess the following logic on ln 788 ensures that?)

Reply:
Yes, you're right, the logic respects specific test values while adding necessary defaults.